apps/chat/conversation.py

```diff
@@ -223,6 +223,11 @@ def truncate_tokens(history, max_token_limit, llm, input_message_tokens):
 
 
 def summarize_history(llm, history, max_token_limit, input_message_tokens, summary, input_messages, pruned_memory):
+    if input_message_tokens >= max_token_limit:
```

Comment:
This reminds me. Now that we have token limits on the models, we can do stuff like prevent users from sending messages where input_message_tokens >= max_token_limit. Shall I make a ticket for this?

Reply:
See https://github.com/dimagi/open-chat-studio/issues/1615 if you have any comments or concerns